packages/art-webpack/src/scripts/build.ts

```diff
@@ -79,8 +83,31 @@ confirmModules(async (answer) => {
     });
 });
 
+function checkVendorsExists() {
+  // TODO 检测vendors文件夹在不在(只判断文件夹还是判断文件夹里面的文件???) 不在则执行art dll(要询问吗???)
+  const checkPath = join(BUILD_PATH, appConfig.get('art:projectVirtualPath'), 'vendors', appConfig.get('art:webpack:dll:version'));
+  const scriptPath = path.resolve(process.cwd(), `./node_modules/art-webpack/dist/scripts/dll.js`);
+  const symlinkPath = path.resolve(__dirname, `../../../art-webpack/dist/scripts/dll.js`);
+  return new Promise((resolve, reject) => {
+    if (!pathExistsSync(checkPath)) {
+      executeNodeScript('node', isDevStage ? symlinkPath : scriptPath).on('close', (code) => {
+        if (code === 0) {
+          resolve(true);
+        }
+      }).on('error', (err) => {
+        // TODO 需要提示自己去执行art dll吗???
+        reject(err);
+      });
+    } else {
+      resolve(true);
+    }
+  });
+}
+
 // Create the production build and print the deployment instructions.
-function build(previousFileSizes: FileSizeProps) {
+async function build(previousFileSizes: FileSizeProps) {
+  const exist = await checkVendorsExists();
+  if (!exist) { return; }
```

Comment:
Check whether vendor file existing, if existing, continue executing following command. If not, run dll if user selected yes, and when dll command done, continue executing following command automatically.